nmostesting/suites/BCP0060101Test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2022 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..GenericTest import GenericTest
+
+NODE_API_KEY = "node"
+CONN_API_KEY = "connection"
+
+
+class BCP0060101Test(GenericTest):
```

Comment:
With hindsight I'm sure we all wish we'd picked something other than adding two digit numbers to the end of the spec numbers to distinguish the test suites of the same specs... "BCP-006-01-01", ugh.